packages/fxa-auth-db-mysql/lib/db/mysql.js

```diff
@@ -1364,6 +1364,16 @@ module.exports = function (log, error) {
       )
   }
 
+  const READ_SECURITY_EVENTS = 'CALL readSecurityEvents_1(?)'
+  MySql.prototype.getSecurityEvents = function (uid) {
+    return this.read(READ_SECURITY_EVENTS, [uid])
+      .then(
```

Comment:
You could even collapse this to `.then(result => result[0])`

Reply:
I followed the style of other methods, I'll do it your way.

Reply:
> You could even collapse this to `.then(result => result[0])`

I think it's better to use ES5 function syntax rather than ES6 arrow functions like other methods.

Reply:
There are some methods below that use fat arrows. I'm all for updating/improving code as we go along.